clients/cobol-lsp-vscode-extension/src/test/suite/copybookanalysis.spec.test.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import * as helper from "./testHelper";
+import * as vscode from "vscode";
+import * as assert from "assert";
+
+suite("TC384131: Analysis Features for copybooks", function () {
+  this.timeout(helper.TEST_TIMEOUT);
+  this.slow(1000);
```

Comment:
```suggestion
  this.slow(6000);
```
There is 5 seconds of waiting.